src/dispatch/alembic/versions/0797208b9eee_.py

```diff
@@ -0,0 +1,85 @@
+"""Adds data model for incident cost
+
+Revision ID: 0797208b9eee
+Revises: 5693756fd192
+Create Date: 2020-06-02 14:30:30.962777
+
+"""
+from alembic import op
+from sqlalchemy.dialects import postgresql
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+# revision identifiers, used by Alembic.
+revision = "0797208b9eee"
+down_revision = "5693756fd192"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "cost",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("details", sqlalchemy_utils.types.json.JSONType(), nullable=True),
+        sa.Column("incident_id", sa.Integer(), nullable=True),
+        sa.Column("created_at", sa.DateTime(), nullable=True),
+        sa.Column("updated_at", sa.DateTime(), nullable=True),
+        sa.ForeignKeyConstraint(["incident_id"], ["incident.id"],),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_table(
+        "pci_disclosure_fines_judgements",
+        sa.Column("id", sa.Integer(), nullable=False),
```

Comment:
Didn't you say these cost ranges are mostly standardized? If so, does it make sense to have them as tables instead of constants? Or are we just allowing flexibility for other industries?

Reply:
Yeah, I'd like to give folks the flexibility of changing them if they want.